test_runner/regress/test_tenant_conf.py

```diff
@@ -107,7 +108,7 @@ def test_tenant_config(neon_env_builder: NeonEnvBuilder):
                     "gc_horizon": 67108864,
                     "gc_period": 80,
                     "image_creation_threshold": 3,
-                    "pitr_interval": 2592000,
+                    "pitr_interval": 60,
```

Comment:
So the point is to make the json file smaller than it was before, to check that the "tail" is truncated away. I get it, knowing the context of issue #3014  and the commit message, but outside that context, no one could guess that that was the point here. Needs a comment at least, or if we could make the file shorter in some more obvious way like having a long comment or option in the beginning and removing it, that would be even better.

Reply:
You are right, added a specific test operation with regards to the short config truncation issue